keyboards/handwired/tractyl_manuform/5x6_right/rpi2040/readme.md

```diff
@@ -0,0 +1,108 @@
+# Schievel's RP2040 Tractyl Manuform (5x6) with a right side trackball
+
+![Imgur](https://i.imgur.com/Xvua5XK.jpeg)
```

Comment:
```suggestion
![Imgur](https://i.imgur.com/Xvua5XKh.jpeg)
```

Reply:
Could you elaborate? The link you suggested results in a 404 for me

Reply:
It uses a smaller image that is better for github markdown.  Not sure why it 404'd for you though.

Reply:
I'm on cell phone right now, maybe that's why. I will test it out later with desktop.

Reply:
yup, works on desktop but not an my iphone. Weird.